src/Microsoft.DotNet.Darc/src/DarcLib/Models/VirtualMonoRepo/AllVersionsPropsFile.cs

```diff
@@ -58,7 +58,9 @@ public void UpdateVersion(string repository, string sha, string packageVersion)
     public bool DeleteVersion(string repository)
     {
         var key = SanitizePropertyName(repository);
-        return Versions.Remove(key + ShaPropertyName) && Versions.Remove(key + PackageVersionPropertyName);
+        var deletedSha = Versions.Remove(key + ShaPropertyName);
+        var deletedVersion = Versions.Remove(key + PackageVersionPropertyName);
+        return deletedSha || deletedVersion;
```

Comment:
FYI you could also use just one variable and do:
```csharp
var deleted = Versions.Remove(key + ShaPropertyName);
deleted |= Versions.Remove(key + PackageVersionPropertyName);
```

Reply:
ok that looks better